util/p9/sres

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
+
+######################################################################################
+# call this script with the name of the result directory you want to copy (i.e. `./res PERF_1555458238`)
```

Comment:
Please make this a usage response that is emitted if the argument is not supplied. See comment for `util/p9/pres`

Reply:
Great suggestion! Updated.